cli/README.md

```diff
@@ -50,18 +50,23 @@ where `<engine>`, `<database>` are the names of some RAI resources to use
-| HTTP retries for RAI sdk in case of errors. (Can be overridden by CLI argument)  | `rai_sdk_http_retries` |
-| A list of containers to use for loading and exporting data.                      | `container`            |
-| The name of the container.                                                       | `container.name`       |
-| The type of the container. Supported types: `local`, `azure`                     | `container.type`       |
-| The path in the container.                                                       | `container.data_path`  |
-| Remote container account                                                         | `container.account`    |
-| Remote container SAS token.                                                      | `container.sas`        |
-| Container for remote container SAS token. (Ex. Azure Blob container)             | `container.container`  |
+| Description                                                                                 | Property               |
+|:--------------------------------------------------------------------------------------------|------------------------|
+| RAI profile.                                                                                | `rai_profile`          |
+| Path to RAI config.                                                                         | `rai_profile_path`     |
+| HTTP retries for RAI sdk in case of errors. (Can be overridden by CLI argument)             | `rai_sdk_http_retries` |
+| A list of containers to use for loading and exporting data.                                 | `container`            |
+| The name of the container.                                                                  | `container.name`       |
+| The type of the container. Supported types: `local`, `azure`, `snowflake`(only data import) | `container.type`       |
+| The path in the container.                                                                  | `container.data_path`  |
+| Remote container account                                                                    | `container.account`    |
+| Remote container SAS token.                                                                 | `container.sas`        |
+| User for remote container SAS token.                                                        | `container.user`       |
+| Password for remote container.                                                              | `container.password`   |
+| User role for remote container. (Ex. Snowflake user role)                                   | `container.role`       |
+| Database for remote container.                                                              | `container.database`   |
+| Schema for remote container.                                                                | `container.schema`     |
+| Warehouse for Snowflake container.                                                          | `container.warehouse`  |
```

Comment:
might be good to generalize the name later, so if we support something else besides SF it makes sense

Reply:
I would keep it as it is to avoid user confusion.